articles/api/authentication/_logout.md

```diff
@@ -58,16 +74,22 @@ Use this endpoint to logout a user. If you want to navigate the user to a specif
 
 1. At the *Configuration* tab, set the fields **Application** (select the application you want to use for the test) and **Connection** (the name of the social connection to use).
 
-1. Copy the <dfn data-key="callback">**Callback URL**</dfn> and set it as part of the **Allowed Logout URLs** of your [Application Settings](${manage_url}/#/applications).
+1. Copy the **_Callback URL_** and set it as part of the **Allowed Logout URLs** of your Application Settings.
 
-1. At the *Other Flows* tab, click **Logout**, or **Logout (Federated)** to log the user out of the identity provider as well.
+1. At the _Other Flows_ tab, click **Logout**, or **Logout (Federated)** to log the user out of the identity provider as well.
 
 
 ### Remarks
 
 - Logging the user out of their identity provider is not common practice, so think about the user experience before you use the `federated` query string parameter.
```

Comment:
I am hesitant to add this. This comment about not doing federated logout applies to social login but not so to enterprise federations. We should add that we refer to social login feds instead.